### PR TITLE
Feature: Implement overriding postgres types using a custom type map

### PR DESCRIPTION
**Pull request by mattiarossi**
Refs https://github.com/divyenduz/plv8ify/issues/27
Implements custom type mapping by
- introducing a new flag (--types-config-file )
- supporting type overrides/additions by including a config file (default, types.ts in the running dir)
- allowing types/interfaces to be imported from other files and still match them against the custom map

### CODE REVIEW

File: src/impl/PLV8ifyCLI.ts [hunk 1]
```diff
@@ -106,6 +125,11 @@ export class PLV8ifyCLI implements PLV8ify {
     const scopedName = this.getScopedName(fn, scopePrefix)
     return `${outputFolder}/${scopedName}.plv8.sql`
   }
+  private getTypeFromMap(type: string) {
+    const typeLocal = type.split('.').pop()
+    //console.log(`Original type: ${type}, Converted type: ${typeLocal} `)
```
Comment (divyenduz) [inline src/impl/PLV8ifyCLI.ts:130]: Remove this `console.log`

**Review by divyenduz — APPROVED** [review]
Broadly looks good, added a comment to remove a `console.log`. Do you mind also adding the new flag to the docs before we merge this!

**Comment by mattiarossi** [comment]
> Do you mind also adding the new flag to the docs before we merge this!

Yes, sorry, the I had it in the first pull request but then I realized it was against the previous version ....

**Comment by divyenduz** [comment]
Thank you for the contribution 🙌🏼 
This is live in [0.0.55](https://github.com/divyenduz/plv8ify/commit/e636dbe003b75836f1c56af9c6bf2d912bd0bdee)

**Comment by divyenduz** [comment]
Btw if you write Fix #27 instead of refs #27, the issue is automatically closed on merge 😁

**Pull request merged**